### PR TITLE
Optimize GFF3 redo (with pseudogene fix)

### PR DESCRIPTION
**Pull request by cmdcolin**
@nathandunn 

Thanks for catching the last bug in #830

This adds a check for pseudogenes and other top level features

### CODE REVIEW

**Comment by cmdcolin** [comment]
Not sure why this build is failing. The error message I see says:

    An error occurred locating the groovy jar under /home/travis/.grails/wrapper/2.4.5/grails-2.4.5/lib/org.codehaus.groovy

Maybe needs to refresh cache. If so maybe someone can do it for me, that would be great (I don't have permissions). Just use the settings button here https://travis-ci.org/GMOD/Apollo/builds/105769296

**Comment by nathandunn** [comment]
K .. restarted it.

**Comment by cmdcolin** [comment]
@nathandunn should be ready now!

**Comment by cmdcolin** [comment]
Actually it is getting mixed up.. The whole thing where it was merged and unmerged made this not have the things it should

**Comment by nathandunn** [comment]
right .   . do you want me to try and test it?

**Comment by cmdcolin** [comment]
Nope...not ready here. It actually doesn't have any of the optimization code on this branch because I did a "git pull origin master" and so it merged the revert into my branch and deleted all the optimizations. I dunno, need to find a way to fix it

**Comment by cmdcolin** [comment]
Okey doke, I was able to "unrevert" this commit (464f55099) and then restore it back to normal and remerge the up to date pseudogene fix

File: grails-app/services/org/bbop/apollo/RequestHandlingService.groovy [hunk 1]
```diff
@@ -63,8 +63,9 @@ class RequestHandlingService {
             MiRNA.class.name,
     ]
 
-    public
-    static List<String> viewableAnnotationList = viewableAnnotationFeatureList + viewableAnnotationTranscriptParentList
+    public static List<String> viewableAlterations = [Deletion.class.name,Insertion.class.name,Substitution.class.name]
```
Comment (nathandunn) [inline grails-app/services/org/bbop/apollo/RequestHandlingService.groovy]: thanks for refactoring this . . much better

**Pull request merged**